src/ArborX_DistributedSearchTree.hpp

```diff
@@ -139,7 +139,9 @@ DistributedSearchTree<MemorySpace, Enable>::DistributedSearchTree(
   MPI_Comm_size(getComm(), &comm_size);
 
   Kokkos::View<Box *, MemorySpace> boxes(
-      Kokkos::ViewAllocateWithoutInitializing("rank_bounding_boxes"),
+      Kokkos::ViewAllocateWithoutInitializing(
+          "ArborX::DistributedSearchTree::DistributedSearchTree::rank_bounding_"
```

Comment:
One of the thing that @dalg24 identified when working on the related PRs is that clang-format sometimes breaks in inopportune places. In situations like this one, it would be better to break after '::' by manually creating a new line ourselves.